DevCycle.SDK.Server.Local/Api/DevCycleLocalClient.cs

```diff
@@ -90,6 +90,11 @@ internal DevCycleLocalClient(
             var platformData = new PlatformData();
             localBucketing.SetPlatformData(platformData.ToJson());
 
+            if(dvcLocalOptions.CdnSlug != "")
+            {
+                logger.LogWarning("The config CDN slug is being overriden, please ensure to update the config to v2 according to the config CDN updates documentation.");
```

Comment:
Might be annoying if theyre actually using the v2 endpoint, maybe add a check if theyre using v1 to log this warning

Reply:
we don't know what endpoint they're pointing to - so better safe than sorry

Reply:
info would possibly get missed - as this can break bucketing expectations i think it's warranted to have a warning

Reply:
we can always change it if someone is seeing more issues